management/src/test/scripts/org/kaazing/gateway/management/jmx/wse.session.with.user.principal.joe.rpt

```diff
@@ -51,10 +51,3 @@ read header "Content-Type" /(?!text\/plain;charset=windows-1252)/
 read header "Connection" "close"
 read [0x01 0x30 0x30 0xFF] # NO-OP
 
-read notify JOE_WSE_SESSION_ESTABLISHED
-
-# read close frames
```

Comment:
Perhaps the intention of these close frames are for when it it is killed that it sends them??